charts/galley/values.yaml

```diff
@@ -23,6 +23,11 @@ config:
     host: aws-cassandra
     replicaCount: 3
   enableFederation: false # keep enableFederation default in sync with brig and cargohold chart's config.enableFederation as well as wire-server chart's tags.federation
+  # Not used if enableFederation is false
```

Comment:
It is not an error to set the helm value, it just gets ignored. If we want to make it an error it would require us to remove this from here and put it in some docs. Which would make it less discoverable.